src/ui/components/map/providers/mapprovider.js

```diff
@@ -90,6 +90,18 @@ export default class MapProvider {
      * @type {boolean}
      */
     this._collapsePins = config.collapsePins || false;
+
+    /**
+     * Locale of the map. Defaults to 'en'
+     * @type {string}
+     */
+    this._locale = config.locale || 'en';
```

Comment:
do we need to set this? if not could just be `const locale =`

Reply:
What do you mean here? Not make locale a private property?

Reply:
yeah, it just looks like this is only used in the constructor and not accessed anywhere else

Reply:
I'll remove the locale property and simply use config.locale directly when setting the language

Reply:
Actually, I'll use const locale because it is more robust.